src/razor/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "ASP.NET Core Razor",
+	"scopeName": "text.aspnetcorerazor",
+	"patterns": [
+		{
+			"include": "text.html.cshtml"
```

Comment:
This causes the `aspnetcorerazor` language to retain VSCode's built-in colorization, but we override everything else.